integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -218,6 +218,12 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
         }
     }
 
+  protected lazy val options: Parser[(String, String)] =
```

Comment:
you can rename `loadOptions` and reuse the same function

Reply:
Done.